code/modules/surgery/organs/stomach/external/wings.dm

```diff
@@ -0,0 +1,230 @@
+///Wing base type. doesn't really do anything
+/obj/item/organ/external/wings
+	zone = BODY_ZONE_CHEST
+	slot = ORGAN_SLOT_EXTERNAL_WINGS
+	layers = EXTERNAL_BEHIND | EXTERNAL_ADJACENT | EXTERNAL_FRONT
+
+	preference = "wings"
+
+/obj/item/organ/external/wings/can_draw_on_bodypart(mob/living/carbon/human/human)
+	if(!human.wear_suit)
+		return TRUE
+	if(human.wear_suit.flags_inv & HIDEJUMPSUIT)
+		if(human.wear_suit.species_exception && is_type_in_list(src, human.wear_suit.species_exception))
+			return TRUE
+		else
+			return FALSE
+	return TRUE
```

Comment:
Okay I might have an actual brain dent, but what about right now? The species_exception checks only ever make sense after checking for HIDEJUMPSUIT, so they need to be in one if-statement or nested. I guess I could also check species_exception first, but that seems like a waste since it only needs to be tested if HIDEJUMPSUIT is true. 

While I doubt it's efficiënt, it does look pretty readable

Reply:
That if that just returns TRUE/FALSE should be turned into just its own `return human.wear_suit.species_exception && is_type_in_list(src, human.wear_suit.species_exception)`, but other than that yes.